driver/ssh.go

```diff
@@ -72,7 +77,7 @@ func (d *SSH) ReadFile(path string) (string, error) {
 }
 
 func (d *SSH) RunCommand(command string) (string, error) {
-	// FIXME: Do we retain client across all command runs?
+	// FIXME: provide refresh interface to somehow refresh d.Client if d.SessionClient somehow gets dropped
```

Comment:
I think we sould retain the client across all commands by the driver. That way reduce the number of connections we have to make. We should probably not call `defer client.Close()` and close all the connections when we're done

Reply:
When exactly do we close the connections